flatdata-py/README.md

```diff
@@ -32,3 +32,21 @@ pip3 install flatdata-py[inspector]  # the inspector feature requires IPython
 flatdata-inspector -p /path/to/my/flatdata.archive
 ```
 
+## Using the writer
+
+`flatdata-writer` is new addition to `flatdata-py`. It helps in creating flatdata files from flatdata schema:
```

Comment:
If you use `new` here, you better also refer to the limitations:
* only bulk-writing (no streaming)
* not perf optimized
* does not allow adding additional sub-archives to existing archive

Reply:
Yes, have to mention limitations
.